examples/node/pdf2svg.js

```diff
@@ -17,32 +20,65 @@ var pdfjsLib = require('pdfjs-dist');
 var pdfPath = process.argv[2] || '../../web/compressed.tracemonkey-pldi-09.pdf';
 var data = new Uint8Array(fs.readFileSync(pdfPath));
 
+var outputDirectory = './svgdump';
+
+try {
+  // Note: This creates a directory only one level deep. If you want to create
+  // multiple subdirectories on the fly, use the mkdirp module from npm.
+  fs.mkdirSync(outputDirectory);
+} catch (e) {
+  if (e.code !== 'EEXIST') {
+    throw e;
+  }
+}
+
 // Dumps svg outputs to a folder called svgdump
-function writeToFile(svgdump, pageNum, callback) {
-  var name = getFileNameFromPath(pdfPath);
-  fs.mkdir('./svgdump/', function(err) {
-    if (!err || err.code === 'EEXIST') {
-      fs.writeFile('./svgdump/' + name + "-" + pageNum + '.svg', svgdump,
-        function(err) {
-          if (err) {
-            console.log('Error: ' + err);
-          } else {
-            console.log('Page: ' + pageNum);
-          }
-          callback();
-        });
-    } else {
-      callback();
-    }
-  });
+function getFilePathForPage(pageNum) {
+  var name = path.basename(pdfPath, path.extname(pdfPath));
+  return path.join(outputDirectory, name + '-' + pageNum + '.svg');
 }
 
-// Get filename from the path
+/**
+ * A readable stream which offers a stream representing the serialization of a
+ * given DOM element (as defined by domstubs.js).
+ *
+ * @param {object} options
+ * @param {DOMElement} options.svgElement The element to serialize
+ */
+function ReadableSVGStream(options) {
+  if (!(this instanceof ReadableSVGStream))
```

Comment:
Nit: add curly brackets around single-line bodies as well for consistency with the other code